core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendConf.scala

```diff
@@ -224,6 +231,12 @@ object SharedBackendConf {
   /** Subnet selector for H2O nodes running inside executors - if the mask is specified then Spark network setup is not discussed. */
   val PROP_NODE_NETWORK_MASK = ("spark.ext.h2o.node.network.mask", None)
 
+  /** Enable or disable stack trace collector on each h2o node */
+  val PROP_NODE_STACK_TRACE_COLLECTOR_ENABLED = ("spark.ext.h2o.stacktrace.collector.enabled", false)
+
+  /** Set how often stack traces are taken on each h2o node */
+  val PROP_NODE_STACK_TRACE_COLLECTOR_INTERVAL = ("spark.ext.h2o.stacktrace.collector.interval", 20)
```

Comment:
What unit is used for the interval? Minutes?

Reply:
Seconds,  should be documented. Thx!